tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -3539,3 +3539,929 @@ Feature: an user gets the resources shared to them
         }
       }
       """
+
+
+  Scenario: user lists the file shared with them from project space
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "new-space" with content "some content" to "testfile.txt"
+    And user "Alice" has sent the following share invitation:
+      | resource        | testfile.txt |
+      | space           | new-space    |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    When user "Brian" lists the shares shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+    """
+    {
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "maxItems": 1,
+          "minItems": 1,
+          "items": {
+            "type": "object",
+            "required": [
+              "@UI.Hidden",
+              "@client.synchronize",
+              "eTag",
+              "file",
+              "id",
+              "lastModifiedDateTime",
+              "name",
+              "parentReference",
+              "remoteItem",
+              "size"
+            ],
+            "properties": {
+              "@UI.Hidden":{
+                "type": "boolean",
+                "enum": [false]
```

Comment:
I am thinkgin of using `const` and not using `type`. two reasons: 1) less schema lines 2) const is best suited for exact comparison

`const` validator is supported now. previously, it wasn't

```suggestion
                "const": false
```